extensions/vertx-http/deployment/src/main/resources/dev-templates/io.quarkus.quarkus-vertx-http/tests.html

```diff
@@ -0,0 +1,257 @@
+{#include main fluid=true}
+
+{#title}Testing{/title}
+
+{#style}
+    #testsInFooter {
+        display: none;
+    }
+    #devUiFooterResizeButton {
+        position: absolute;
+        right: 50vw;
+    }
+    .main-container{
+        margin-top: 0px !important;
+    }
+    .testsFooterButton i::after {
+       font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans","Liberation Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
+        font-weight: normal;
+    }
+    .btnTestsGuide i::after {
+        content: " Guide";
+    }
+    .btnRerunAllTests i::after {
+        content: " Run all";
+    }
+    .btnRerunFailedTests i::after {
+        content: " Run failed";
+    }
+    .btnBrokenOnlyTests i::after {
+        content: " Toggle broken only mode";
+    }
+    .btnPrintFailuresTests i::after {
+        content: " Print failures";
+    }
+    .btnToggleTestOutput i::after{
+        content: " Toggle test output";
+    }
+    .btnToggleInstrumentationBasedReload i::after {
+        content: " Toggle instrumentation based reload";
+    }
+    .btnTestsResults{
+        display: none;
+    }
+    
+{/style}
+
+{#script}
+    var testPageUrl = window.location.protocol + "//" + window.location.host + testsPathname.substr(0, testsPathname.indexOf('/dev/')) + "/dev/io.quarkus.quarkus-vertx-http/tests";
```

Comment:
Same here for the `{urlbase}`

Reply:
urlbase is available in extension that include tiles, but not in this one, that is more internal.... (part of Dev UI)